frontend/src/js/components/subscription/SubscriptionDrawer.tsx

```diff
@@ -151,7 +151,8 @@ export const SubscriptionDrawer = (props: SubscriptionDrawerProps) => {
       />
     </div>
   );
-  const cardDetailsDisabled = isTrial && !billing;
+  const hasBilling = (billing && Object.keys(billing).length > 0);
```

Comment:
```suggestion
  const hasBilling = !isEmpty(billing);
```
with 
```
import { isEmpty } from '@northern.tech/utils/helpers';
```
added at the top